recipe/spyder-menu.json

```diff
@@ -109,103 +54,7 @@
-                                "com.apple.xcode.pascal-source",
-                                "com.apple.xcode.strings-text",
-                                "com.apple.xcode.tcsh-script",
-                                "com.apple.xcode.yacc-source",
-                                "com.apple.xcode.zsh-script",
-                                "com.apple.xml-property-list",
-                                "com.netscape.javascript-source",
-                                "com.scenarist.closed-caption",
-                                "com.sun.java-source",
-                                "com.sun.java-web-start",
-                                "net.daringfireball.markdown",
-                                "org.khronos.glsl-source",
-                                "org.oasis-open.xliff",
-                                "public.ada-source",
-                                "public.assembly-source",
-                                "public.bash-script",
-                                "public.c-header",
-                                "public.c-plus-plus-header",
-                                "public.c-plus-plus-source",
-                                "public.c-source",
-                                "public.case-insensitive-text",
-                                "public.comma-separated-values-text",
-                                "public.csh-script",
-                                "public.css",
-                                "public.delimited-values-text",
-                                "public.dylan-source",
-                                "public.filename-extension",
-                                "public.fortran-77-source",
-                                "public.fortran-90-source",
-                                "public.fortran-95-source",
-                                "public.fortran-source",
-                                "public.html",
-                                "public.json",
-                                "public.ksh-script",
-                                "public.lex-source",
-                                "public.log",
-                                "public.m3u-playlist",
-                                "public.make-source",
-                                "public.mig-source",
-                                "public.mime-type",
-                                "public.module-map",
-                                "public.nasm-assembly-source",
-                                "public.objective-c-plus-plus-source",
-                                "public.objective-c-source",
-                                "public.opencl-source",
-                                "public.pascal-source",
-                                "public.patch-file",
-                                "public.perl-script",
-                                "public.php-script",
-                                "public.plain-text",
                                 "public.python-script",
```

Comment:
```suggestion
                                "public.plain-text",
                                "public.python-script"
```

So, I would retain at least plain-text for macOS. We could retain them all, or add them at the request of users. On macOS, these do not make them default. The absence of them results in the application not included in "Recommended Applications" for opening the file.

<img width="802" alt="Screenshot 2024-08-27 at 9 34 57 PM" src="https://github.com/user-attachments/assets/fff49eb2-3f87-4088-84ed-61ec16edd1c3">